tokio/src/io/mod.rs

```diff
@@ -211,6 +211,11 @@ cfg_io_driver_impl! {
         pub use driver::{Interest, Ready};
     }
 
+    cfg_aio! {
+        mod poll_aio;
+        pub use poll_aio::{AioSource, PollAio, PollAioEvent};
+    }
+
```

Comment:
You can move this outside the `cfg_io_driver_impl!` block like `AsyncFd` is. Also, this adds a module similar to the `unix` module that `AsyncFd` is in.
```suggestion
cfg_aio! {
    mod poll_aio;

    pub mod freebsd {
        pub use super::poll_aio::{AioSource, PollAio, PollAioEvent};
    }
}
```

Reply:
done.